src/client/ui/panels/HierarchyPanelContainer.scss

```diff
@@ -29,3 +30,14 @@
   padding: 4px 5px;
 }
 
+:local(.conflictDisplay) {
+  bottom: 2%;
+  left: 50%;
+  transform: translate(-50%, 0%);
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+  flex-wrap: wrap;
+  position: absolute;
+  width: 80%;
```

Comment:
Hmm, I'm not sure about using percentages. I think we use fixed pixels everywhere else.